Client/mods/deathmatch/logic/lua/CLuaFunctionParseHelpers.cpp

```diff
@@ -581,6 +580,23 @@ ADD_ENUM(WING_AIRTRAIL, "wing_airtrail")
 ADD_ENUM(VEH_GUN, "veh_gun")
 IMPLEMENT_ENUM_END("vehicle-dummy")
 
+IMPLEMENT_ENUM_BEGIN(eCollisionShapes)
+ADD_ENUM(COLLISION_BOX, "box")
+ADD_ENUM(COLLISION_SPHERE, "sphere")
+ADD_ENUM(COLLISION_TRIANGLE, "triangle")
+ADD_ENUM(COLLISION_VERTEX, "vertex")
+IMPLEMENT_ENUM_END("collision-shape")
+
+IMPLEMENT_ENUM_BEGIN(eCollisionKey)
+ADD_ENUM(COLLISION_KEY_SIZE, "size")
+ADD_ENUM(COLLISION_KEY_MATERIAL, "material")
+ADD_ENUM(COLLISION_KEY_POSITION, "position")
+ADD_ENUM(COLLISION_KEY_RADIUS, "radius")
+ADD_ENUM(COLLISION_KEY_LIGHTING, "lighting")
+ADD_ENUM(COLLISION_KEY_VERTICES, "vertices")
+ADD_ENUM(COLLISION_KEY_MOVE, "move")
+IMPLEMENT_ENUM_END("collision-key")
```

Comment:
New code should not introduce warnings. VS 2019 complains about the use of regular enums. You should prefer `enum class`es.

Here is an example commit on how to do that: https://github.com/multitheftauto/mtasa-blue/pull/1055/commits/0adfc8b288f0a495f9b6d5d6647a8c3af896e4aa